<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCaseDialog</class>
 <widget class="QDialog" name="NewCaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новая задача</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="CaseGroupBox">
     <property name="title">
      <string>Задача</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="CaseNameLabel">
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="CaseDescriptionLabel">
        <property name="text">
         <string>Описание:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="CaseDescriptionLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="CaseNameEdit">
        <property name="text">
         <string>Новая задача</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Тип игрового автомата</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="TemplateComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Выберите тип автомата</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Автомат 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Автомат 2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Каталог отчёта</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="DirectoryLineEdit">
        <property name="text">
         <string>Выберите каталог</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DirectoryPushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="CreateCasePushButton">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelPushButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
